src/node.cc

```diff
@@ -2456,44 +2456,46 @@ static void GetParentProcessId(Local<Name> property,
 
 
 static Local<Object> GetFeatures(Environment* env) {
-  EscapableHandleScope scope(env->isolate());
+  Isolate* isolate = env->isolate();
+  EscapableHandleScope scope(isolate);
+  Local<Boolean> true_value = True(isolate);
+  Local<Boolean> false_value = False(isolate);
 
-  Local<Object> obj = Object::New(env->isolate());
+  Local<Object> obj = Object::New(isolate);
 #if defined(DEBUG) && DEBUG
-  Local<Value> debug = True(env->isolate());
+  Local<Boolean> debug = true_value;
 #else
-  Local<Value> debug = False(env->isolate());
+  Local<Boolean> debug = false_value;
```

Comment:
Instead of making this change multiple times, what if we instead removed the temporary variables instead and just set the values directly in the `obj->Set()` calls?